kuroko2.gemspec

```diff
@@ -26,6 +26,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'retryable'
   s.add_dependency 'faraday'
   s.add_dependency 'sprockets'
+  s.add_dependency 'slim'
```

Comment:
Is this intended? It seems we already have slim-rails and it claims "there is no need to include gem "slim" in your Gemfile.": https://github.com/slim-template/slim-rails